app/src/main/java/com/avito/android/krop/demo/MainActivity.kt

```diff
@@ -115,7 +123,7 @@ class MainActivity : AppCompatActivity() {
                     .load("file:///android_asset/default_image.jpg")
                     .error(R.drawable.image)
                     .centerInside()
-                    .resize(1024, 1024)
+                    .resize(4000, 4000)
```

Comment:
This can lead to out of memory. Check this carefully.
Also, demo image is 1920p.

Reply:
Yeah, I see. It was required for test purposes.
I don't regard it as a problem now, although I'll keep an eye on it.